From 9c07b186f3fd23294335c1a76c77437018673747 Mon Sep 17 00:00:00 2001
From: "Liu,Dalei" <daleix.liu@intel.com>
Date: Tue, 12 Aug 2014 14:26:40 +0800
Subject: [PATCH 145/164] atomisp.h: Add element hts in atomisp data struct

Add a new element "hts" in the struct atomisp_sensor_mode_data. The
HAL need hts for flicker program.

Change-Id: I7da5572f2d161411b9a5c99fb9caeb10804f8a0e
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-7690
Signed-off-by: Liu,Dalei <daleix.liu@intel.com>
---
 include/linux/atomisp.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/atomisp.h b/include/linux/atomisp.h
index 142b565..545a508 100644
--- a/include/linux/atomisp.h
+++ b/include/linux/atomisp.h
@@ -679,7 +679,7 @@ struct atomisp_sensor_mode_data {
 	unsigned int output_height;
 	uint8_t binning_factor_x; /* horizontal binning factor used */
 	uint8_t binning_factor_y; /* vertical binning factor used */
-	uint8_t reserved[2];
+	uint16_t hts;
 };
 
 struct atomisp_exposure {
-- 
1.7.9.5

